www/js/model.js

```diff
@@ -259,14 +276,40 @@ define([
 			},
 			setColor: function(/*string*/ id, /*string*/ color){
 				this.getNode(id).color = color;
-			}
+			},
+			setAuthorStatus: function(/*string*/ id, /*string*/ part, /*string*/ status){
+				// Summary: function to set the status of node editor in author mode.
+				if(!this.getNode(id).authorStatus){
```

Comment:
Ok I saw the calls. What are the values the authorStatus take, is it for completeness or something?